### PR TITLE
Shared runtime config

### PR DESCRIPTION
**Pull request by Rick-Kirkham**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[43d8d8f](https://github.com/OfficeDev/office-js-docs-pr/commits/43d8d8fcc332ac2b1917f208d8986418187756fe)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/configure-your-add-in-to-use-a-shared-runtime.md](https://github.com/OfficeDev/office-js-docs-pr/blob/share-runtime-config/docs/excel/configure-your-add-in-to-use-a-shared-runtime.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/configure-your-add-in-to-use-a-shared-runtime?branch=pr-en-us-2117) | [Details](#user-content-063dae14-3d237c46)

<a id="063dae14-3d237c46"></a>
### [docs/excel/configure-your-add-in-to-use-a-shared-runtime.md](https://github.com/OfficeDev/office-js-docs-pr/blob/share-runtime-config/docs/excel/configure-your-add-in-to-use-a-shared-runtime.md)
 - **Line 97, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C24%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202008241830205879-2117%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=b73U7uiuU8kpRBzem1d%2BBQ9kilTQmA%2B%2FCX%2BSbgEHeCw%3D&st=2020-08-24T18%3A25%3A56Z&se=2020-09-24T18%3A30%3A56Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/excel/configure-your-add-in-to-use-a-shared-runtime.md [hunk 1]
```diff
@@ -88,7 +88,24 @@ Follow these steps for a new or existing project to configure it to use a shared
    ...
    ```
 
-8. Save your changes and rebuild the project.
+8. Make sure the taskpane.html has a `<script>` tag that references the dist/functions.js file. The following is an example:
+
+```html
+<script type=”text/javascript” src=”/dist/functions.js” ></script>
+```
```
Comment (ElizabethSamuel-MSFT) [inline docs/excel/configure-your-add-in-to-use-a-shared-runtime.md:95]: This should address the build suggestion
```suggestion
    ```html
    <script type=”text/javascript” src=”/dist/functions.js” ></script>
    ```
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[43fc6e3](https://github.com/OfficeDev/office-js-docs-pr/commits/43fc6e386d7404701aaf2dad403c2c9f2f1188fd)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/configure-your-add-in-to-use-a-shared-runtime.md](https://github.com/OfficeDev/office-js-docs-pr/blob/share-runtime-config/docs/excel/configure-your-add-in-to-use-a-shared-runtime.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/configure-your-add-in-to-use-a-shared-runtime?branch=pr-en-us-2117) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C24%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202008241833471637-2117%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=MwWLNUVuLRv%2FsPcuzW8xWxg%2FmqymKkfbW2ajfDpavck%3D&st=2020-08-24T18%3A29%3A22Z&se=2020-09-24T18%3A34%3A22Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[d17793f](https://github.com/OfficeDev/office-js-docs-pr/commits/d17793fafe7efbea83af8e084c829632ef43de95)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/configure-your-add-in-to-use-a-shared-runtime.md](https://github.com/OfficeDev/office-js-docs-pr/blob/share-runtime-config/docs/excel/configure-your-add-in-to-use-a-shared-runtime.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/configure-your-add-in-to-use-a-shared-runtime?branch=pr-en-us-2117) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C24%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202008241837276048-2117%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=cTJ45puD25d9bm%2FCgY%2F5BURO76RAuIcTLH8eBBFiC8M%3D&st=2020-08-24T18%3A33%3A15Z&se=2020-09-24T18%3A38%3A15Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/excel/configure-your-add-in-to-use-a-shared-runtime.md (hunk 1 above)
Comment (Rick-Kirkham) [reply docs/excel/configure-your-add-in-to-use-a-shared-runtime.md]: fixed

File: docs/excel/configure-your-add-in-to-use-a-shared-runtime.md [hunk 2]
```diff
@@ -88,7 +88,24 @@ Follow these steps for a new or existing project to configure it to use a shared
    ...
```
Comment (ElizabethSamuel-MSFT) [inline docs/excel/configure-your-add-in-to-use-a-shared-runtime.md:88]: Update ms.date?

File: docs/excel/configure-your-add-in-to-use-a-shared-runtime.md [hunk 3]
```diff
@@ -88,7 +88,24 @@ Follow these steps for a new or existing project to configure it to use a shared
...
    ```
 
-8. Save your changes and rebuild the project.
+8. Make sure the taskpane.html has a `<script>` tag that references the dist/functions.js file. The following is an example:
+
+   ```html
+   <script type=”text/javascript” src=”/dist/functions.js” ></script>
```
Comment (ElizabethSamuel-MSFT) [inline docs/excel/configure-your-add-in-to-use-a-shared-runtime.md:94]:
```suggestion
   <script type="text/javascript" src="/dist/functions.js" ></script>
```

File: docs/excel/configure-your-add-in-to-use-a-shared-runtime.md [hunk 4]
```diff
@@ -88,7 +88,24 @@ Follow these steps for a new or existing project to configure it to use a shared
    ...
    ```
 
-8. Save your changes and rebuild the project.
+8. Make sure the taskpane.html has a `<script>` tag that references the dist/functions.js file. The following is an example:
```
Comment (ElizabethSamuel-MSFT) [inline docs/excel/configure-your-add-in-to-use-a-shared-runtime.md:91]:
```suggestion
8. Make sure the taskpane.html has a `<script>` tag that references the dist/functions.js file. The following is an example.
```

File: docs/excel/configure-your-add-in-to-use-a-shared-runtime.md [hunk 5]
```diff
@@ -88,7 +88,24 @@ Follow these steps for a new or existing project to configure it to use a shared
...
+
+   ```html
+   <script type=”text/javascript” src=”/dist/functions.js” ></script>
+   ```
+
+   > [!NOTE]
+   > If the add-in uses Webpack and the HtmlWebpackPlugin to insert script tags, as add-ins created by the Yeoman generator do (see [Create the add-in project](#create-the-add-in-project) above), then you must ensure that the functions.js module is included in the `chunks` array as in the following example:
```
Comment (ElizabethSamuel-MSFT) [inline docs/excel/configure-your-add-in-to-use-a-shared-runtime.md:98]:
```suggestion
   > If the add-in uses Webpack and the HtmlWebpackPlugin to insert script tags, as add-ins created by the Yeoman generator do (see [Create the add-in project](#create-the-add-in-project) above), then you must ensure that the functions.js module is included in the `chunks` array as in the following example.
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[82c4fb4](https://github.com/OfficeDev/office-js-docs-pr/commits/82c4fb47c4409444c229d847f2cafeb862344a30)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/configure-your-add-in-to-use-a-shared-runtime.md](https://github.com/OfficeDev/office-js-docs-pr/blob/share-runtime-config/docs/excel/configure-your-add-in-to-use-a-shared-runtime.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/configure-your-add-in-to-use-a-shared-runtime?branch=pr-en-us-2117) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C25%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202008251702047901-2117%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=6kwaNJjJfntyMc8HJA1e65WZi6QDintfNIG6x9vLCAc%3D&st=2020-08-25T16%3A57%3A40Z&se=2020-09-25T17%3A02%3A40Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[1c47a9d](https://github.com/OfficeDev/office-js-docs-pr/commits/1c47a9d81947ce50ad38ead31b4592acd3a453b6)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/configure-your-add-in-to-use-a-shared-runtime.md](https://github.com/OfficeDev/office-js-docs-pr/blob/share-runtime-config/docs/excel/configure-your-add-in-to-use-a-shared-runtime.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/configure-your-add-in-to-use-a-shared-runtime?branch=pr-en-us-2117) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C25%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202008251703520301-2117%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=kGUx578YplSQvCTFYy4Wi%2F%2Bbs6%2BZAIHbBN9n7aB3c7k%3D&st=2020-08-25T16%3A59%3A29Z&se=2020-09-25T17%3A04%3A29Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Pull request merged**